Hypercube/src/app.php

```diff
@@ -13,10 +15,18 @@
 $app->register(new ServiceControllerServiceProvider());
 $app['hypercube.controller'] = function () use ($config) {
     return new HypercubeController(
-        new TesseractService($config['executable'])
+        new CmdExecuteService(),
```

Comment:
Needs logger.  Maybe we can feed it a monolog service using http://silex.sensiolabs.org/doc/2.0/providers/monolog.html?  I believe it conforms to the PSR logging interface.